src/main/java/com/stripe/model/ApplicationFee.java

```diff
@@ -1,193 +1,240 @@
+  public void setOriginatingTransactionObject(Charge expandableObject) {
+    this.originatingTransaction =
+        new ExpandableField<Charge>(expandableObject.getId(), expandableObject);
   }
 
-  // <editor-fold desc="list">
   /**
-   * List all application fees.
+   * Returns a list of application fees you’ve previously collected. The application fees are
+   * returned in sorted order, with the most recent fees appearing first.
    */
   public static ApplicationFeeCollection list(Map<String, Object> params) throws StripeException {
-    return list(params, null);
+    return list(params, (RequestOptions) null);
   }
 
   /**
-   * List all application fees.
+   * Returns a list of application fees you’ve previously collected. The application fees are
+   * returned in sorted order, with the most recent fees appearing first.
    */
   public static ApplicationFeeCollection list(Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    return requestCollection(classUrl(ApplicationFee.class), params, ApplicationFeeCollection.class,
-        options);
+    String url = String.format("%s%s", Stripe.getApiBase(), "/v1/application_fees");
+    return requestCollection(url, params, ApplicationFeeCollection.class, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="retrieve">
   /**
-   * Retrieve an application fee.
+   * Retrieves the details of an application fee that your account has collected. The same
+   * information is returned when refunding the application fee.
    */
   public static ApplicationFee retrieve(String id) throws StripeException {
-    return retrieve(id, (RequestOptions) null);
+    return retrieve(id, (Map<String, Object>) null, (RequestOptions) null);
   }
 
   /**
-   * Retrieve an application fee.
+   * Retrieves the details of an application fee that your account has collected. The same
+   * information is returned when refunding the application fee.
    */
   public static ApplicationFee retrieve(String id, RequestOptions options) throws StripeException {
-    return retrieve(id, null, options);
+    return retrieve(id, (Map<String, Object>) null, options);
   }
 
   /**
-   * Retrieve an application fee.
+   * Retrieves the details of an application fee that your account has collected. The same
+   * information is returned when refunding the application fee.
    */
-  public static ApplicationFee retrieve(String id, Map<String, Object> params,
-      RequestOptions options) throws StripeException {
-    return request(RequestMethod.GET, instanceUrl(ApplicationFee.class, id), params,
-        ApplicationFee.class, options);
+  public static ApplicationFee retrieve(
+      String id, Map<String, Object> params, RequestOptions options) throws StripeException {
+    String url =
+        String.format("%s%s", Stripe.getApiBase(), String.format("/v1/application_fees/%s", id));
+    return request(ApiResource.RequestMethod.GET, url, params, ApplicationFee.class, options);
```

Comment:
5) Retrieve (standard): Static method to retrieve a resource with a given ID

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/ApplicationFee.java#L230-L239